packages/principal/test/ed25519.spec.js

```diff
@@ -6,6 +6,8 @@ import { varint } from 'multiformats'
 describe('signing principal', () => {
   const { Signer } = Lib
 
+  /** @type {Lib.Signer.EdSigner} */
```

Comment:
Is this type assertion doin anything? I'm not sure what line it asserts about given the newline before and after